CHANGELOG.md

```diff
@@ -1,213 +1,106 @@
-  - Update to latest legs and update version
-  - don't be as aggressive about using :3000 / :3001 on client request (#254) ([filecoin-project/storetheindex#254](https://github.com/filecoin-project/storetheindex/pull/254))
-  - Create creating-an-index-provider.md (#253) ([filecoin-project/storetheindex#253](https://github.com/filecoin-project/storetheindex/pull/253))
-  - add cors headers to find interface (#252) ([filecoin-project/storetheindex#252](https://github.com/filecoin-project/storetheindex/pull/252))
-  - Update dependencies and version
-  - Reduce log volume and log progress (#246) ([filecoin-project/storetheindex#246](https://github.com/filecoin-project/storetheindex/pull/246))
-  - Indexer auto-syncs after period of no contact with publisher (#247) ([filecoin-project/storetheindex#247](https://github.com/filecoin-project/storetheindex/pull/247))
-  - Only increase error metric on error
-  - Remove readBeforeErr hack now that cbor-gen bug is fixed
+# 1.17.0-rc2 / 2022-06-29
+
+This is the second release candidate of the upcoming OPTIONAL release Lotus v1.17.0. This release also supports the Filecoin network v16 Skyr upgrade (FVM) with some small bug fixes, new features & improvements. Full changelog will be published upon final release.
+
+## Mainnet nv16 Skyr Upgrade
+
+This release candidate sets mainnet upgrade at epoch 1044660, 2022-06-16T17:30:00Z. All node operators, including storage providers, must upgrade to v1.16.0 (or a later release) before that time. Storage providers must update their daemons, miners, market and worker(s).
+
+Your lotus node will switch from the Legacy VM (that depended on go-based specs-actors) to Filecoin Virtual Machine FVM (that uses Rust-based builtin-actors) atomically upon the upgrade.
+
+The Skyr upgrade introduces the following FIPs, enhancements and bug fixes, delivered in [built-actors v8](https://github.com/filecoin-project/builtin-actors/releases/tag/v8.0.0) and [ref-fvm v1.0.0](https://github.com/filecoin-project/ref-fvm/releases/tag/fvm%40v1.0.0):
+
+- [FIP-0030](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0030.md)
+- [FIP-0031](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0031.md)
+- [FIP-0032](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0032.md))
+- [FIP-0027](https://github.com/filecoin-project/FIPs/blob/master/FIPS/fip-0027.md)
+- [Bug fix for the ProveReplicaUpdates method](https://github.com/filecoin-project/builtin-actors/pull/138)
+- [New proofs version for SnarkPack](https://github.com/filecoin-project/builtin-actors/pull/474/commits/3027c365f516e1cba6f156d4fb9dbd8c893d5b62)
+
+## New Features
+
+- feat: worker: lotus-worker run --no-default ([filecoin-project/lotus#8672](https://github.com/filecoin-project/lotus/pull/8672))
+- feat: sched: Per worker concurrent task count limits ([filecoin-project/lotus#8725](https://github.com/filecoin-project/lotus/pull/8725))
```

Comment:
would be nice to add a sub point on how to set

Reply:
https://github.com/filecoin-project/lotus/blob/f46cfb44b97190e183d568a96cbe39a791cc9f0a/CHANGELOG.md?plain=1#L217-L218